lms/envs/production.py

```diff
@@ -1168,3 +1168,6 @@
 
 SUPPORT_DESK_EMAILS = ENV_TOKENS.get('SUPPORT_DESK_EMAILS')
 INSTALLED_APPS.append('openedx.features.ucsd_features')
+
+if ENV_TOKENS.get('COUNTRIES_ELIGIBLE_FOR_DISCOUNTS'):
```

Comment:
Do we need this check.

Reply:
Yes otherwise the default value will be overwritten